logger.Dockerfile

```diff
@@ -29,11 +27,10 @@ LABEL description="Sidecar to output Cassandra system logs to stdout"
 COPY ./config/logger/vector_config.toml /etc/vector/vector.toml
 COPY --from=builder /usr/lib64/libstdc++.so.6 /usr/lib64/libstdc++.so.6
 COPY --from=builder /usr/bin/vector /usr/bin/vector
-COPY --from=builder --chown=999:999 /var/lib/vector /var/lib/vector
```

Comment:
Question #1: Does removing the ownership change here cause issues in OpenShift when running as non-root user?

If you have tested this change in OpenShift and there is no issue when running with restricted security context, the rest of this is moot.

If we do use `chown` in the COPY here, I have found that for permissions to work correctly in OpenShift, you need to create the user with UID:GID in the `builder` stage, and then either do it again in this final stage, or copy the /etc/passwd, /etc/group and /etc/shadow files from the `builder` stage after creating the user in that stage.

 For some strange reason, it seems if you create files in a previous stage where the `cassandra` user does not exist, and then copy them in the final stage, even if you use a RUN line to change ownership and permissions on the files you copied, it still somehow does not have the correct permissions for running in OpenShift where the effective UID and GID are some random ids that are not `999`. Only when I created the `cassandra` user in the previous stage and then created that user again in the final stage did the permissions get set the way I expected/intended.

Question #2: All of that said, I'm guessing removing this line is fine because `var/lib/vector` will only be added to the image as a mount and won't have the OpenShift permissions issues we had with some of the Management API images?

Reply:
Yes, it's removed here because Kubernetes creates it. Adding chown here would make no difference.